source/MRViewer/MRUISaveChangesPopup.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <string>
+#include <functional>
+
+#include "exports.h"
+
+namespace MR
+{
+
+namespace UI
+{
+
+struct SettingsSaveChangesPopup
```

Comment:
we usually name settings structures reversed:
```suggestion
struct SaveChangesPopupSettings